architecture/additional_concepts/authorization.adoc

```diff
@@ -490,6 +490,77 @@ JSON merge on RBAC resources.
 endif::[]
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 
+### Aggregated ClusterRoles
+
+As of 1.9, ClusterRoles can be created by combining other ClusterRoles using an `aggregationRule`. The
```

Comment:
1.9 -> 3.9

Reply:
Which product versions is this available in? It can't just be 3.9.